src/views/AddTodo.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+<div class="add-todos">
+    <div class="container w-full lg:w-1/2 mx-auto text-left">
+        <h1 class="text-blue-500 my-8 font-bold text-2xl text-center">ADD TODO : </h1>
+      <form class="lg:w-1/2 w-full mx-auto lg:px-6 lg:py-10 p-4 bg-slate-100 shadowm-sm rounded-lg" @submit.prevent="addTodo">
+        <div class="mb-8 text-left text-xl">
+            <label class="mb-2 block text-gray-800 font-medium ">Category Name:</label>
+            <input type="name" class="bg-slate-50 text-black text-opacity-50 cursor-not-allowed rounded-md border border-gray-200 p-3 focus:outline-none w-full uppercase focus:ring-1 focus:ring-red-300"  disabled v-model="form.categoryName">
```

Comment:
This should be a dropdown with all the categories of the current user. Also the `todos` table should store `category_id` referencing the category and not the category name itself.

Reply:
okay sir